edx/edx-enterprise
# [ENT-432] Remove EnterpriseIntegratedChannel model.

The `EnterpriseIntegratedChannel` model was unused and it seems unnecessary.

**JIRA:** [ENT-432](https://openedx.atlassian.net/browse/ENT-432)

**Merge deadline:** End of sprint V.

**Testing instructions:**

This code was unused, so this should not change any functionality.

1. Make sure the migration occurs correctly.

**Merge checklist:**

  **DO NOT** just add dependencies to `requirements/*.txt` files.

**Post merge:**
